juniper_official/Solutions/Icmp-outlier/icmp.rule

```diff
@@ -0,0 +1,252 @@
+/*
+ * Sends ICMP probes to user specified destination host and notifies anomalies 
+ * when round trip time average response is above static or dynamic thresholds.
+ * And also detects outlier across the devices in device group ML algorithms.
+ * Three inputs control detection
+ * 
+ *   1) host_var, is a destination IP or host name where ICMP probes send 
+ *      periodically.
+ *   2) count_var, is a ICMP ping count. By default count is set to 1. 
+ *   3) rtt-threshold-var, is the threshold that causes the rule to report an
+ *      anomaly. By default it is 1000 micro seconds. This rule will set a
```

Comment:
Can we have rtt-threshold-var defaults in milli seconds rather than micro seconds?

Reply:
Done added UDF